src/components/Dapp.re

```diff
@@ -928,4 +928,4 @@ let make = () => {
       <DefaultLook isGqlLoaded />
     </Rimble.Box>
   </Rimble.Flex>;
-};
+};
```

Comment:
I'm going to find a way to save the vscode settings in the repo so this doesn't keep happening...

Reply:
I think the `.vscode` files work